server/src/main/java/org/opensearch/action/admin/cluster/stats/ClusterStatsNodes.java

```diff
@@ -854,4 +858,48 @@ public XContentBuilder toXContent(final XContentBuilder builder, final Params pa
 
     }
 
+    static class PeerRecoveryStats implements ToXContentFragment {
+
+        private long totalStartedRecoveries;
+        private long totalFailedRecoveries;
+        private long totalCompletedRecoveries;
+        private long totalRetriedRecoveries;
+        private long totalCancelledRecoveries;
+
+        PeerRecoveryStats(List<NodeStats> nodeStats) {
+            for (NodeStats nodeStat : nodeStats) {
+                if (nodeStat.getPeerRecoveryStats() != null) {
+                    totalStartedRecoveries += nodeStat.getPeerRecoveryStats().getTotalStartedRecoveries();
+                    totalFailedRecoveries += nodeStat.getPeerRecoveryStats().getTotalFailedRecoveries();
+                    totalCompletedRecoveries += nodeStat.getPeerRecoveryStats().getTotalCompletedRecoveries();
+                    totalRetriedRecoveries += nodeStat.getPeerRecoveryStats().getTotalRetriedRecoveries();
+                    totalCancelledRecoveries += nodeStat.getPeerRecoveryStats().getTotalCancelledRecoveries();
```

Comment:
Throughout the repo, we usually support this behaviour via an `add()` method in the respective `*Stats` class. Example: https://github.com/opensearch-project/OpenSearch/blob/53c0ce38b26569c90f8980f1c2b0b70e5ebbf6fc/server/src/main/java/org/opensearch/index/translog/RemoteTranslogStats.java#L156-L167